packages/react/src/index.ts

```diff
@@ -81,13 +81,33 @@ export * from './SwatchColorPicker';
 export * from './TeachingBubble';
 export * from './Text';
 export * from './TextField';
-export * from './Theme';
 export * from './ThemeGenerator';
 export * from './TimePicker';
 export * from './Toggle';
 export * from './Tooltip';
 export * from './Utilities';
 export * from './WeeklyDayPicker';
 export * from './WindowProvider';
+/**
+ * Now explicitly declaring Theme exports that are NOT already being exported from Styles.
+ * Styles and Theme both exported the same names which causes conflicting
+ * star exports with webpack5. See here: https://github.com/microsoft/fluentui/issues/21601.
+ */
+export * from './utilities/ThemeProvider/index';
+export {
+  CommunicationColors,
+  DefaultSpacing,
+  Depths,
+  FluentTheme,
+  LocalizedFontFamilies,
+  LocalizedFontNames,
+  mergeThemes,
+  MotionDurations,
+  MotionTimings,
+  MotionAnimations,
+  NeutralColors,
+  SharedColors,
+} from './Theme';
+export type { ComponentStyles, ComponentsStyles, PartialTheme, Theme } from './Theme';
```

Comment:
Should you export these from the `./Theme` file itself and still do `export *` here?

Reply:
Ideally these should only be exported from Theme.ts OR Style.ts, not both. However removing them from one of those would be a breaking change. This was the workaround I suggested.